blog/_posts/2017-09-07-team_code_hoppers_blog.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Our journey - with rgsoc and Open Source!
+layout: post
+created_at: Thursday Sep 08 2017
+permalink: blog/2017-09-07-team_code_hoppers_blog
+current: blog
+author: roselyne
+twitter: code_Hoppers
+categories:
+- student-posts
+---
+
+Being at #rgsoc has been one of the most fruitful yet challenging experiences this year. Every single day was exciting and came with it’s own set of learning curve.
+
+#### Our accomplishments so far?
+
+In a nutshell, we learnt about coding, teamwork, collaboration and issue resolution. We even got to learn how to effectively communicate with remote teams across the world, listening and focusing on a common goal, defining tasks and keeping schedules. Well, sometimes not everything went as we had planned and from that we learnt that nothing is exactly perfect and we always have to keep striving to efficiently communicate and resolve our blockers. **It has been an amazing journey!**
+
+#### Our take from the summer!
+
+- We learned about [Rails](http://rubyonrails.org/) and [Elastic search](http://rubyonrails.org/)
+- Keeping and maintaining coding standards specified by an organisation
+- Learnt the importance of implementing peer to peer reviews
+- We felt the need of contributing to a shared vision
+- We revamped the search functionality by designing the UI and backend
+- We added new features to the new landing page - Cynthia helped alot with this
+- Simon helped us implement the backend [searchKick](https://github.com/ankane/searchkick) functionality
+
+![Cynthia's take!](/img/blog/2017/team_code_hoppers_cynthia.jpg)<font color="grey"><small><i>Rgsoc has been a great experience. I have learnt so much. It has been fun collaborating with my partner, the coaches and our mentor. I hope to continue contributing to open farm</i></small></font>
```

Comment:
Image credits are missing over here. Can you please add the image credits using image credits class?